relay-config/src/config.rs

```diff
@@ -1014,6 +1014,18 @@ pub struct Processing {
     /// Maximum rate limit to report to clients.
     #[serde(default = "default_max_rate_limit")]
     pub max_rate_limit: Option<u32>,
+    /// True if normalization shouldn't run for events coming from internal relays. Defaults to `false`.
+    #[serde(default)]
+    pub disable_normalization: bool,
```

Comment:
Should we flip this to `enable_normalization` with default `true`?

Reply:
Merge two configs into an enum in [83894b6](https://github.com/getsentry/relay/pull/3377/commits/83894b69eefca3bb456cb850fe2ac84decff41e1).